src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  selectedPersonName?: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ selectedPersonName = '' }) => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(peopleFromServer => {
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code